ci-operator/config/openshift/ci-secret-mirroring-controller/openshift-ci-secret-mirroring-controller-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+binary_build_commands: make install
+images:
+- dockerfile: |
```

Comment:
Shouldn't this be in the repo?

Reply:
https://github.com/openshift/ci-secret-mirroring-controller/pull/4